modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1102,9 +1245,10 @@ PERF_TEST_P_(AddWeightedPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
-    EXPECT_EQ(out_mat_gapi.size(), sz_in);
-
+    {
+        EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
+        EXPECT_EQ(out_mat_gapi.size(), sz);
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.

Reply:
Applied, thanks